.github/scripts/scraper.py

```
@@ -11,145 +13,212 @@
 logger.remove()
 logger.add(lambda msg: print(msg, end=""), colorize=True, format="<green>{time:YYYY-MM-DD at HH:mm:ss}</green> | <level>{level: <8}</level> | <cyan>{message}</cyan>")
 
+# --- 取得元設定 ---
+# arXiv は RSS(export.arxiv.org/rss/cs.AI) が間欠的に0件を返すため、Atom API に寄せる
+# （ai_mail_digest の paper_sources.py と同じ取得手法。submittedDate 降順で週末でも空にならない）。
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)